pkg/kopia/command/common.go

```diff
@@ -15,16 +15,48 @@
 package command
 
 import (
+	"os"
+
 	"github.com/kanisterio/kanister/pkg/field"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/logsafe"
 )
 
+const (
+	// LogLevelVarName is the environment variable that controls datastore log level.
+	LogLevelVarName = "DATA_STORE_LOG_LEVEL"
+	// FileLogLevelVarName is the environment variable that controls datastore file log level.
+	FileLogLevelVarName = "DATA_STORE_FILE_LOG_LEVEL"
+)
+
+func NonEmptyOrDefault[T comparable](t T, def T) T {
+	var empty T
+	if t != empty {
+		return t
+	}
+	return def
+}
+
+func GetEnvOrDefault(name, def string) string {
+	return NonEmptyOrDefault(os.Getenv(name), def)
+}
+
+// LogLevel will return either value from env or "error" as default value
+func LogLevel() string {
+	return GetEnvOrDefault(LogLevelVarName, LogLevelError)
+}
+
+// FileLogLevel will return value from env
+func FileLogLevel() string {
+	return os.Getenv(FileLogLevelVarName)
```

Comment:
what does this default to?

Reply:
empty string, and if it is empty, we won't pass this flag at all (as it was before my changes)
```
	if fileLogLevel != "" {
		c = c.AppendLoggableKV(fileLogLevelFlag, fileLogLevel)
	}
```